core/services/business-decision-point/services/PoliciesService.js

```diff
@@ -36,10 +36,12 @@ class PoliciesService {
       matchingPolicies.map(async (policy) => {
         const singleDecision = await isConditionMet(policy.condition, req.body);
         if (singleDecision === true) {
-          excludes.push({
-            priority: policy.priority,
-            fields: policy.excludes,
-          });
+          if (typeof policy.excludes !== "undefined") {
```

Comment:
@jbuese it's better to explicitly check if `excludes` is an array, e.g. `Àrray.isArray(policy.excludes)`. The value could be `null` for some reason.